functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -122,12 +146,35 @@ Function Get-DbaSqlBuildReference {
 			
 			$LastUpdated = Get-Date -Date $result.LastUpdated
 			if ($LastUpdated -lt (Get-Date).AddDays(-45)) {
-				Write-Message -Level Warning -Silent $Silent -Message "Index is stale, last update on: $(Get-Date -Date $LastUpdated -Format s)"
+				Write-Message -Level Warning -Silent $Silent -Message "Index is stale, last update on: $(Get-Date -Date $LastUpdated -Format s), try the -Update parameter to fetch the most up to date index"
 			}
 			
 			$result.Data | Select-Object @{ Name = "VersionObject"; Expression = { [version]$_.Version } }, *
 		}
 		
+		function Get-DbaSqlBuildReferenceIndexOnline {
+			[CmdletBinding()]
+			Param (
+				[bool]
+				$Silent
+			)
+			$url = 'https://sqlcollaborative.github.io/assets/dbatools-buildref-index.json'
```

Comment:
Should be a configuration entry. Information should not be hardcoded. For example, large organizations could provide an internal path and thus allow machines without online access to update.
I'd recommend creating a new configuration file "assets", then place the config there and call it with something like this:
`Get-DbaConfigValue -Name 'assets.sqlbuildreference'`